server/src/main/java/org/opensearch/snapshots/SnapshotsService.java

```diff
@@ -338,16 +338,6 @@ public ClusterState execute(ClusterState currentState) {
                 }
 
                 boolean remoteStoreIndexShallowCopy = REMOTE_STORE_INDEX_SHALLOW_COPY.get(repository.getMetadata().settings());
-                Version minVersionInCluster = currentState.nodes().getMinNodeVersion();
-                if (remoteStoreIndexShallowCopy && !minVersionInCluster.onOrAfter(Version.V_2_9_0)) {
-                    remoteStoreIndexShallowCopy = false;
-                    logger.warn(
-                        "Setting {} is ignored as some of the nodes in cluster are on version older than {}. Minimum node version in cluster is: {}",
-                        REMOTE_STORE_INDEX_SHALLOW_COPY,
-                        Version.V_2_9_0,
-                        minVersionInCluster
-                    );
-                }
```

Comment:
I think we should still have this check here. Users can move cluster to 2.9 apply the setting at repo level and then move back to 2.8 since 2.8 and 2.9 are compatible with each other. This way putRepo will not be invoked but setting can still remain.